profile/README.md

```diff
@@ -1,16 +1,13 @@
 # Single sign-on is coming to the B.C. government’s GitHub organizations 
 
-🔔 We're enabling single sign-on (SSO) for GitHub. We’re starting with the bcgov GitHub organization. After we finish transitioning bcgov, we’ll enable other OCIO-managed GitHub organizations. 
+:sparkles: The Developer Experience team has enabled Single Sign-On (SSO) for GitHub. We’re starting with the bcgov GitHub organization. After we finish transitioning bcgov, we’ll enable other OCIO-managed GitHub organizations.
 
-To keep access to the bcgov GitHub organization, all users must link their IDIRs to their GitHub IDs between June 30th and July 30th, 2024. If users don’t complete this by July 30th, 2024, they’ll be automatically removed from the bcgov GitHub organization, and by extension, all the code repositories connected to bcgov. 
+By **July 30, 2024**, all bcgov GitHub organization users must link their IDIRs with their GitHub IDs. If this isn’t done by the deadline, access to the GitHub organization will be automatically removed. 
 
-Contractors without IDIRs must get one to before they can enable SSO. Learn how to order an IDIR. 
+Contractors without IDIRs must order one before SSO can be enabled.
 
-[Learn more about the timelines and instructions](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github?utm_source=main&utm_medium=github&utm_campaign=bcgov-readme#single-sign-on-is-coming-to-the-bc-governments-github-organizations).
+[Learn more about IDIRs and technical information.](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide?utm_source=guide-idir&utm_medium=github&utm_campaign=bcgov-readme#idirs)
 
-For more information, please see our transition guides:
-- [Transition guide for IDIRs](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide?utm_source=guide-idir&utm_medium=github&utm_campaign=bcgov-readme#idirs)
-
--  [Transition guide for GitHub](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github?utm_source=main&utm_medium=github&utm_campaign=bcgov-banner#single-sign-on-is-coming-to-the-bc-governments-github-organizations)
+[Learn more about the timelines and instructions.](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/#the-changes)
```

Comment:
https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/bc-government-organizations-in-github/?utm_source=main&utm_medium=github&utm_campaign=bcgov-readme#the-changes